server/middleware/__tests__/configureEstablishment.spec.js

```diff
@@ -0,0 +1,41 @@
+const { configureEstablishment } = require('../configureEstablishment');
+
+describe('configureEstablishment', () => {
+  const next = jest.fn();
+
+  beforeEach(() => {
+    next.mockClear();
+  });
+
+  it('should use the set the session from the config', () => {
+    const req = {
+      session: {
+        establishmentName: 'wayland',
+        establishmentId: 793,
+      },
+    };
```

Comment:
Is it worth running a precedent test, selection of the appropriate establishmentName from session over header? This could be incorporated into this test by providing erroneous header data?

Reply:
meh, ignore this, I miss understood the code 👍